intermediate_source/ddp_tutorial.rst

```diff
@@ -19,13 +19,33 @@ every module replica, i.e., no module replication within a process. The code in
 this tutorial runs on an 8-GPU server, but it can be easily generalized to
 other environments.
 
+Comparison between ``DataParallel`` and ``DistributedDataParallel``
+-------------------------------------------------------------------
+
+Before we dive in, let's clarify why, despite the added complexity, you would
+consider using ``DistributedDataParallel`` over ``DataParallel``, remembering
+that **model parallel** (covered in the
```

Comment:
`DataParallel` does not work with model parallel yet. For now, model parallel can only be combined with ``DistributedDataParallel``.